browser/brave_browser_main_extra_parts.cc

```diff
@@ -30,4 +35,20 @@ void BraveBrowserMainExtraParts::PreMainMessageLoopRun() {
   // So, call it after browser threads are created.
   g_brave_browser_process->brave_widevine_bundle_manager()->StartupCheck();
 #endif
+  // TODO(iefremov): Maybe find a better place for this initialization.
+  g_brave_browser_process->brave_p3a_service()->Init();
+
+  // Record default values for some histograms because we want these stats to be
+  // uploaded anyways. Corresponding components will write new values according
+  // to their usage scenarios.
+  if (first_run::IsChromeFirstRun()) {
+    RecordImporterP3A(importer::ImporterType::TYPE_UNKNOWN);
+  }
+  brave_shields::RecordShieldsUsageP3A(brave_shields::kNeverClicked,
```

Comment:
this doesn't seem right, shouldn't it also be in `first_run::IsChromeFirstRun`?

Reply:
The name is misleading, renamed to `MaybeRecord...`